src/hotspot/share/gc/g1/g1_globals.hpp

```diff
@@ -379,6 +379,16 @@
            number of free regions and the expected survival rates in each   \
            section of the heap.")                                           \
                                                                             \
+  product(uint, G1RemoveSelfForwardPtrsWorkerCost, 2, DIAGNOSTIC,           \
```

Comment:
Looking at how it's used, `G1RemoveSelfForwardPtrsWorkerCost` is essentially #workers per evac-fail regions. The current name/doc msg exposes too much impl detail and is not end-user friendly. How about "G1PerRetainedRegionThreads: The number of worker threads used for each retained (evacuation failure) region, subject to ParallelGCThreads constraint"?